hystrix-contrib/hystrix-reactivesocket-event-stream/src/main/java/com/netflix/hystrix/contrib/reactivesocket/BasePayloadSupplier.java

```diff
@@ -0,0 +1,19 @@
+package com.netflix.hystrix.contrib.reactivesocket;
+
+import com.fasterxml.jackson.dataformat.cbor.CBORFactory;
+import io.reactivesocket.Payload;
+import rx.Observable;
+import rx.subjects.BehaviorSubject;
+
+import java.util.function.Supplier;
+
+public abstract class BasePayloadSupplier implements Supplier<Observable<Payload>> {
+    protected final CBORFactory jsonFactory;
```

Comment:
maybe - CBORFactory inherits from jsonFactory though and you can replace it with the JsonFactory if you want.